charts/graph-network-indexer/templates/indexer-service/deployment.yaml

```diff
@@ -75,7 +75,7 @@ spec:
             - name: INDEXER_SERVICE_DATABASE__POSTGRES_URL
               value: 'postgres://$(POSTGRES_USERNAME):$(POSTGRES_PASSWORD)@{{ index $values.postgresConfig "host" }}:{{ index $values.postgresConfig "port" }}/{{ index $values.postgresConfig "database" }}'
           command:
-          - /usr/local/bin/service
+          - {{ $values.command | default "/usr/local/bin/service" }}
```

Comment:
The default comes from the key having a default in values.yaml, and shouldn't get hardwired here.